src/kvstore/RocksEngine.cpp

```diff
@@ -15,6 +15,9 @@
 
 DEFINE_bool(move_files, false,
             "Move the SST files instead of copy when ingest into dataset");
+DEFINE_uint32(readahead_size, 512 * 1024,
```

Comment:
Maybe just use the default value of rocksdb, it has a default value of 256KB. see https://github.com/facebook/rocksdb/pull/3282. Also it won't help a lot, our bottleneck is the `Seek` operation.

Reply:
We need a benchmark here

Reply:
You could give a benchmark here. It only works for hdd. Block in ssd is not consecutive, so prefetch won't help.

Reply:
> You could give a benchmark here. It only works for hdd. Block in ssd is not consecutive, so prefetch won't help.

I agree with @critical27 , This parameter has an advantage for HDDs, but has a performance impact on SSDs because it has other overhead when turned on.

Reply:
ok  I'll close it firstly